lti_consumer/lti_xblock.py

```diff
@@ -1357,10 +1337,20 @@ def _get_context_for_template(self):
         allowed_attributes = dict(bleach.sanitizer.ALLOWED_ATTRIBUTES, **{'img': ['src', 'alt']})
         sanitized_comment = bleach.clean(self.score_comment, tags=allowed_tags, attributes=allowed_attributes)
 
-        # Set launch handler depending on LTI version
-        lti_block_launch_handler = self.runtime.handler_url(self, 'lti_launch_handler').rstrip('/?')
-        if self.lti_version == 'lti_1p3':
-            lti_block_launch_handler = self.runtime.handler_url(self, 'lti_1p3_launch_handler').rstrip('/?')
+        if self.lti_version == 'lti_1p1':
+            # Set launch handler depending on LTI version
+            lti_block_launch_handler = self.runtime.handler_url(self, 'lti_launch_handler').rstrip('/?')
+        else:
+            # Runtime import since this will only run in the
+            # Open edX LMS/Studio environments.
+            from lti_consumer.api import get_lti_1p3_launch_start_url  # pylint: disable=import-outside-toplevel
```

Comment:
If this import requires LMS/Studio environment, should we move this import to ``compat.py``?

Reply:
@shimulch This doesn't need any LMS API, but when run inside the XBlock's test environment it'll fail because workbench is not allowed to access Django imports, so any internal `api.py` calls from the XBlock needs to be imported at runtime. 

There's a similar usage of this in this file already, here: https://github.com/edx/xblock-lti-consumer/blob/master/lti_consumer/lti_xblock.py#L838